doc/userguide/lua/lua-functions.rst

```diff
@@ -81,131 +81,6 @@ For detection, use the specific buffer (cf :ref:`lua-detection` for a complete l
       return needs
   end
 
-HttpGetRequestBody and HttpGetResponseBody.
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Make normalized body data available to the script through
-HttpGetRequestBody and HttpGetResponseBody.
-
-There no guarantees that all of the body will be available.
-
-Example:
-
-::
-
-  function log(args)
-      a, o, e = HttpGetResponseBody();
-      --print("offset " .. o .. " end " .. e)
-      for n, v in ipairs(a) do
-          print(v)
-      end
-  end
-
-HttpGetRequestHost
```

Comment:
Am I missing something, or have we missed documenting `request_host()` in the new HTTP section for Lua libs?

Reply:
Added with https://github.com/OISF/suricata/pull/13378